src/main/java/racingcar/view/input/dto/RacingRequest.java

```diff
@@ -0,0 +1,55 @@
+package racingcar.view.input.dto;
+
+import racingcar.common.exception.dto.ExceptionEnum;
+import racingcar.view.input.exception.InputException;
+
+
+/**
+ * @author : jiffyin7@gmail.com
+ * @since : 24. 10. 25.
+ */
+public record RacingRequest(
+    String carNames,
+    int roundCount
+) {
+
+  private static boolean hasWhiteSpace(String text){
+    return text.chars().anyMatch(Character::isWhitespace);
+  }
+
+  private static int validateRoundCount(String roundCount){
+    if (roundCount == null || roundCount.trim().isEmpty()){
```

Comment:
isBlank를 사용할 수 있을 것 같아요!
```suggestion
    if (roundCount == null || roundCount.isBlank()){
```

Reply:
String의 좋은 메서드네요! 코드가 한결 간단해집니다.